presentation/handler/bill_handler.go

```diff
@@ -18,52 +18,43 @@ func NewBillHandler(billService *IBillService, groupService *IGroupService) *Bil
 	return &BillHandler{billService: *billService, groupService: *groupService}
 }
 
-// GetByID 		func get bill by id
+// GetByID 		gets bill by id.
 //
 //	@Summary	Get Bill by ID
 //	@Tags		Bill
 //	@Accept		json
 //	@Produce	json
-//	@Param		id	path		string	true	"Bill Id"
+//	@Param		id	path		string	true	"Bill ID"
 //	@Success	200	{object}	dto.GeneralResponseDTO{data=dto.BillOutputDTO}
 //	@Router		/api/bill/{id} [get]
 func (h BillHandler) GetByID(c *fiber.Ctx) error {
-	id := c.Params("id")
-	if id == "" {
-		return core.Error(c, fiber.StatusBadRequest, fmt.Sprintf(ErrMsgParameterRequired, "id"))
-	}
-	bid, err := uuid.Parse(id)
+	id, err := uuid.Parse(c.Params("id"))
 	if err != nil {
-		return core.Error(c, fiber.StatusBadRequest, fmt.Sprintf(ErrMsgParseUUID, id, err))
+		return core.Error(c, fiber.StatusBadRequest, fmt.Sprintf(ErrMsgParseUUID, c.Params("id"), err))
```

Comment:
Why did you remove the check whether a parameter is included? If we now forget the parameter we receive: "Could not parse uuid: item, error: invalid UUID length: 4". I think the previous error msg that the parameter ID is required is more suitable here.